sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -649,6 +649,14 @@ object SQLConf {
       .doubleConf
       .createWithDefault(0.05)
 
+  val SHOW_STATS_IN_EXPLAIN =
```

Comment:
Why not doing this by default? Do we need an extra flag?

If needed, the name should be `SHOW_TABLE_STATS_IN_EXPLAIN`